local/protocol/locate.go

```diff
@@ -0,0 +1,34 @@
+package protocol
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+// Utilities to deal with Scoot instances.
```

Comment:
In the broader sense of what protocol can entail, yes it fits. However there is the 'wire protocol' and then there is all the other stuff usually passed as flags or env vars, which I typically categorize as 'config'. I think it's ok to leave it this way though.